src/native/fs/fs_napi.cpp

```diff
@@ -77,7 +77,8 @@ struct FSWorker : public Napi::AsyncWorker
     std::string _backend;
     std::string _desc;
     int _errno;
-    int thresholdInMilliSec = 2;
+    protected:
```

Comment:
on second thought, no need to go into protected visibility.
this is a struct, so all is public by definition unless otherwise specified.
I would simply remove the protected keyword and it should be fine.